使用 connect-button 组件报错

```tsx
      <ConnectButton
          type="primary"
          icon={<WalletWhiteColorful />}
          account={{
              address: '3ea2cfd153b8d8505097b81c87c11f5d05097c18',
          }}
      />
```
报错
```js
ERROR in ./node_modules/@ant-design/web3/dist/esm/connect-modal/index.js 90:0-69
Module not found: Error: Can't resolve '../../../common/src/utils/warning.js' in 'frontend/node_modules/@ant-design/web3/dist/esm/connect-modal'
```

--- a/.changeset/shy-plants-deliver.md
+++ b/.changeset/shy-plants-deliver.md
@@ -1,0 +1,5 @@
+---
+'@ant-design/web3-common': minor
+---
+
+fix:connect-button warning

--- a/packages/common/src/utils/index.ts
+++ b/packages/common/src/utils/index.ts
@@ -2,3 +2,4 @@ export * from './request';
 export * from './format';
 export * from './test-utils';
 export * from './createGetBrowserLink';
+export * from './warning';

